main/src/ca/uwaterloo/flix/language/phase/typer/TypeReduction.scala

```diff
@@ -156,29 +159,64 @@ object TypeReduction {
     val candidateMethods = clazz.getMethods
       .filter(m => m.getName == method)
```

Comment:
Would it be cleaner to have an `isCandidate` function?

Reply:
I’ll do this, thank you.